Site/LogMan.io/Release Notes/v25.28.md

```diff
@@ -0,0 +1,1008 @@
+# Release Notes v25.28
+
+## Patches
```

Comment:
i suggest reversing the patch order so that the latest entry is always at the top